cmd/minishift/cmd/util/util.go

```diff
@@ -71,3 +72,17 @@ func UnpackAddons(addonsDir string) error {
 
 	return nil
 }
+
+func ShowProgressDots(progressDots chan bool) {
+	go func() {
+		for {
+			select {
+			case <-progressDots:
+				return
+			default:
+				fmt.Print(".")
+				time.Sleep(5 * time.Second)
```

Comment:
Maybe configurable... 5 seconds between a dot is quite long. 

The general human response time to think something is acted on is about 300ms. So, printing needs to happen fast before it starts a long process... but it is possible to take about 3 seconds before the brain needs another stimulus. 5 stretches it... so in the case of the registration it might need more ? but for starting the VM it can be faster...

WDYT? what gave the best results?

Reply:
3 seconds looks good. Configurable make sense. For registration, it takes long so will keep three and for starting VM 2 second probably.